### DISCUSSION TITLE
Schema Validator Error: The property noopener is not recognized by the schema (e.g. schema.org) for an object of type Article.

### DISCUSSION THREAD

**Discussion by btakita** [schemaorg/schemaorg#3485] [Q&A]
Hi everyone, I ran the schema validator on rdfa markup that has an `<a target=_blank rel=noopener>` tag in an Article. The validator gives the error:

```
The property noopener is not recognized by the schema (e.g. schema.org) for an object of type Article.
```

Is this a problem with the validator? Since `rel=noopener` or `rel="noopener noreferrer"` fixes a security vulnerability when `target=_blank`. Is there a workaround?

**Comment by corrideat** [comment]
I hope the response isn't too late, but a solution is using a prefix. E.g.,

```html
<html prefix="schema: https://schema.org">
<a href="https://x.com" rel="noopener" rel="noopener schema:YOURSCHEMAPROPERTY" href="http://example.com">test</a>
</html>
```

**Reply by corrideat** [reply]
(you'd use the prefix instead of `vocab`). E.g., `<div typeof="schema:example">` instead of `<div vocab="https://schema.org" typeof="example">`
